src/Signal.lua

```diff
@@ -74,10 +76,14 @@ function Signal:connect(callback)
 	local listener: Listener = {
 		callback = callback,
 		disconnected = false,
```

Comment:
You can use an if-expression for both of these assignments.

Reply:
I don't think we can, at least not without changing our CI to use Luau instead of Lua 5.X. Darklua doesn't seem to handle converting if-exp to regular if else statements.